plugins/rapid7_vulndb/komand_rapid7_vulndb/util/extract.py

```diff
@@ -94,6 +107,14 @@ class Content:
         "references",
     ]
 
+    @classmethod
+    @Util.retry(tries=1, timeout=30, exceptions=PluginException, backoff_seconds=1)
+    def retrieve_by_identifier(cls, identifier: str):
+        # extract data from API
+        response = requests.get(urljoin(cls.content_url, identifier))
+        _response_error_handler(response.status_code, response.text)
+        return response.json()
```

Comment:
Is `JSONDecodeError` handled? I cant see it

Reply:
Not currently but can be added to next update per conversation.